.gitignore

```diff
@@ -0,0 +1 @@
+/.idea
```

Comment:
Don't ignore the idea folder - some stuff underneath should be written to source control